content/200-concepts/300-more/050-environment-variables/200-using-multiple-env-files.mdx

```diff
@@ -7,38 +7,75 @@ tocDepth: 2
 
 <TopBlock>
 
-There is a risk that your production database could be deleted if you store different connection URLs to each of your environments within a single `.env` file.
+When developing an application you will generally have different environments such as `development`, `test` and `production` which you use to **separate the stages of the development cycle**.
 
-One solution is to have multiple `.env` files which each represent different environments. In practice this means you create a file for each of your environments:
+Prisma can access each environment via a [connection url](../../../reference/database-reference/connection-urls) which is stored in a `.env` file.
+
+</TopBlock>
+
+## Scenario: Running integration tests
+
+You have three [connection urls](../../../reference/database-reference/connection-urls) for each of your environments stored in a single `.env` file, each one connects to a different Postgres database.
+
+```env file=.env
+
+// Connection URL to development environment
+DEVELOPMENT_DATABASE_URL="postgresql://prisma:prisma@localhost:5432/development"
+
+// Connection URL to testing environment
+TESTING_DATABASE_URL="postgresql://prisma:prisma@localhost:5432/test"
+
+// Connection URL to production environment
+PRODUCTION_DATABASE_URL="postgresql://prisma:prisma@pg-prod.xxxx.rds.amazonaws.com.com:5432/production"
+
+```
+
+<Admonition type="warning">
+
+`PRODUCTION_DATABASE_URL` is added above for clarity in explaining the scenario. It's **not recommended** to store your production credentials locally, even if they are git ignored.
+
+</Admonition>
+
+You are running your integration test inside a [docker container](../../../guides/testing/integration-testing#add-docker-to-your-project) which, upon startup:
+
+1. Creates a database
+1. Runs a migration with your application schema
+1. Seeds the database with fake data
+1. Runs the tests
+1. **Deletes all the data in the database**
+
+The docker container is passed an environment variable from the `.env` file which specifies which database it should use. 
+
+There is a risk that you accidentally pass the **wrong** environment variable to the docker container and **all the data is deleted** once the tests are finished running.
```

Comment:
How do you chose which env var is passed to the docker Container from the .env file? Usually the env file is relevant in the context you run the CLI or Prisma Client - not via a Docker container. Can you illustrate the example a bit more so I can understand what you are doing?